pkg/integrations/v2/blackbox_exporter/blackbox_exporter.go

```diff
@@ -18,7 +20,7 @@ var DefaultConfig = Config{
 type Config struct {
 	BlackboxConfigFile string                             `yaml:"config_file,omitempty"`
 	BlackboxTargets    []blackbox_exporter.BlackboxTarget `yaml:"blackbox_targets"`
-	BlackboxConfig     blackbox_config.Config             `yaml:"blackbox_config,omitempty"`
+	BlackboxConfig     util.RawYAML                       `yaml:"blackbox_config,omitempty"`
```

Comment:
the changes in this file match the changes we already did the integrations v1 to enable to converters to pass the raw yaml to the river output